lib/tool_shed/webapp/api2/repositories.py

```diff
@@ -508,7 +508,5 @@ def get_readmes(
         return readmes(self.app, repository, changeset_revision)
 
 
-def _hack_fastapi_4428(as_dict) -> dict:
-    # https://github.com/tiangolo/fastapi/pull/4428#issuecomment-1145429263
-    # after pydantic2 swap to really returning the object
-    return RepositoryMetadata(__root__=as_dict).dict()["__root__"]
+def _hack_fastapi_4428(as_dict) -> RepositoryMetadata:
+    return RepositoryMetadata(root=as_dict)
```

Comment:
This can probably just be dropped since it's called only in one place?

Reply:
Good call, thank you!